ra_aid/llm.py

```
@@ -47,19 +50,25 @@ def initialize_llm(provider: str, model_name: str, temperature: float | None = N
             temperature=temperature if temperature is not None else 0.3,
             model=model_name,
         )
+    elif provider == "gemini":
+        return ChatGoogleGenerativeAI(
+            api_key=os.getenv("GEMINI_API_KEY"),
+            model=model_name,
+            **({"temperature": temperature} if temperature is not None else {})
+        )
     else:
         raise ValueError(f"Unsupported provider: {provider}")
```

Ruff (0.8.2)

Avoid specifying long messages outside the exception class

(TRY003)